common/source/docs/common-matekf765-wse.rst

```diff
@@ -106,6 +106,21 @@ With this option, :ref:`SERIAL7_PROTOCOL<SERIAL7_PROTOCOL>` must be set to "23",
 
 Any UART can be used for RC system connections in ArduPilot also, and is compatible with all protocols except PPM. See :ref:`common-rc-systems` for details.
 
+If using the Fport with a Frsky receiver you should consider having it on te version 2.1.0 FCC, the newest 2.1.1 seems to have telemtry issues
+
+Parameters:
+
+Use RX6 pin to connect your FPort pin. The configuration should be:
```

Comment:
this is not required...its an alternate setup....the setup using the TX pin is the normal connection and not doing a SWAP since F4 autopilots cannot even do that...

Reply:
yes but it's a f7 flight controller

Reply:
i just tried the other pin without the swap and it's not working..

Reply:
I had Matek try it and it works for them using the current instruction for the RX6 pin....I have asked the dev team for another confirmation one way or the other....it works for me on a 765-wing also